### ISSUE TITLE
[Error] Can't export app installed by crossover inside distrobox

### ISSUE THREAD

**Issue by zihadio** [89luca89/distrobox#392]
I am using Fedora Silverblue. I installed Ubuntu 22.04 inside distrobox with separate home. I installed Crossover 22. And installed winrar in it. everything works. But I can't export the app. Can you help me?
What I want is to open a zip file using winrar from an app exported by distrobox.

**Comment by 89luca89** [comment]
Hi @zihadio 
where does Crossover put the desktop entries for the windows apps you install with it?

If it's not in a standard folder, I'm afraid distrobox will not be able to find it

**Comment by zihadio** [comment]
@89luca89 I created ubuntu box with seperate home. So, crossover installed the .desktop in `.local/share/applications/` inside distrobox.

![Screenshot_২০২২০৯০১_২০১২১৭](https://user-images.githubusercontent.com/19417232/187935610-493aa2f6-b84f-4af0-858d-b4e505b94f90.png)

**Comment by 89luca89** [comment]
Cool thanks, and when you use the export command, what is the output of the command?
Can you post the command you launch with the output?

**Comment by 89luca89** [comment]
Remember names are case sensitive
From my test, exporting like:

`distrobox-export --app WinRAR`

works

**Comment by zihadio** [comment]
@89luca89 this is what I get
```
❯ distrobox-export --app WinRAR
Error: cannot find any desktop files.
Error: trying to export a non-installed application.

An error occurred
```

**Comment by 89luca89** [comment]
@zihadio can you try with 1.3.2 from git?
I've tried and it worked for me

**Comment by 89luca89** [comment]
Tested it again and is something fixed in 1.3.2+, so I'll close this as it's resolved
